tests/python/common/transforms/test_transform.py

```diff
@@ -2492,6 +2492,17 @@ def test_module_add_self_loop(idtype):
     assert "w1" in new_g.edges["plays"].data
     assert "w2" in new_g.edges["follows"].data
 
+    # Case5: add self-loops for a homogeneous graph
```

Comment:
What's the difference between Case 5 and Case 1?

Also, maybe move this up to Case 2 since it is similar to Case 1 and renumber the others.

Reply:
I have changed the title and close PR 5856. 
This case is similar to case1. However, our test should cover the example in doc. So I add this case for test_transform.py.